x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/content.test.tsx

```diff
@@ -42,4 +42,16 @@ describe('AttacksPageContent', () => {
       expect(screen.getByTestId('schedule')).toBeInTheDocument();
     });
   });
+
+  it('should render `Connector` filter', async () => {
```

Comment:
Leaving the same comment I put in your other [PR](https://github.com/elastic/kibana/pull/244575), but feel free to ignore it: I feel like this test and the previous one should be handled within the very first one (should render correctly). Checking that things on the page are rendered is exactly what the first tests is doing...